qdrant/v1.2.x/how_to.md

```diff
@@ -533,6 +533,20 @@ client.update_collection(
 )
 ```
 
+### Appendable mmap
```

Comment:
how-to/appendable-mmap doesn't really match the semantics of this section

Reply:
I've updated it. Is this better?

https://github.com/qdrant/docs/blob/428ab37d56ce6a41e033331fa965c27a03e308f9/qdrant/v1.2.x/how_to.md?plain=1#L536-L554